app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -433,22 +434,25 @@ struct DeviceDelegateOculusVR::State {
         trackpadTouched = (controllerState.inputState.Touches & ovrTouch_Joystick) != 0;
         trackpadX = controllerState.inputState.Joystick.x;
         trackpadY = controllerState.inputState.Joystick.y;
-        axes[0] = trackpadX;
-        axes[1] = -trackpadY; // We did y axis intentionally inverted in FF desktop as well.
+        const int32_t kNumAxes = 4;
+        float axes[kNumAxes];
+        axes[(uint32_t)ImmersiveAxis::TouchpadX] = axes[(uint32_t)ImmersiveAxis::TouchpadY] = 0.0f;
+        axes[(uint32_t)ImmersiveAxis::ThumbstickX] = trackpadX;
+        axes[(uint32_t)ImmersiveAxis::TouchpadY] = -trackpadY; // We did y axis intentionally inverted in FF desktop as well.
```

Comment:
I assume this should be `ImmersiveAxis::ThumbstickY`?

Reply:
I also not fond of all the casting. The consts should of the correct type if possible. Even if we have to change some APIs.